src/scheduled_tasks.rs

```diff
@@ -398,6 +474,13 @@ async fn active_counts(pool: &mut DbPool<'_>) {
   }
 }
 
+/// Re-calculate all the active counts
+async fn all_active_counts(pool: &mut DbPool<'_>, intervals: [(&str, &str); 4]) {
+  for i in intervals {
+    active_counts(pool, i).await;
+  }
+}
```

Comment:
I dont see the point of this function, you can keep the loop directly in active_counts() like before.

Reply:
This is actually the main time saver of this PR. The six_months query is *unavoidably* > 5 minutes. Even if I fix the locking issue, a 5 minute query still shouldn't be run every hour.

Reply:
Ah I missed that, makes sense.